config/test.js

```diff
@@ -0,0 +1,18 @@
+const test = {
+  network: {
+    name: 'Test',
+    rest: ['https://api.n06.testnet.vega.xyz', 'https://api.n07.testnet.vega.xyz'],
```

Comment:
I find this overriding hard to read. Much simpler:

```suggestion
const mockPort = 9090

const test = {
  // See below comment why i have put this here
  test: {
     // Renamed to remove node as node doesn't really add any extra description
     mockPort,
  },
  network: {
    name: 'Test',
    rest: [`http://localhost:${mockPort}`],
```